build.gradle.kts

```diff
@@ -47,13 +52,22 @@ dependencies {
     mappings("de.oceanlabs.mcp:mcp_stable:22-1.8.9")
     forge("net.minecraftforge:forge:1.8.9-11.15.1.2318-1.8.9")
 
+    // Add essential dependency so we can access elementa (Gui creator)
```

Comment:
We avoided dependencies like this so far, can this be done without including all of essential?